ogr/services/gitlab/project.py

```diff
@@ -361,6 +362,8 @@ def get_branches(self) -> list[str]:
 
     def get_file_content(self, path, ref=None) -> str:
         ref = ref or self.default_branch
+        # GitLab cannot resolve './'
+        path = os.path.normpath(path)
```

Comment:
I was looking for this function, but everything I managed to find on the `Path` class was impure /o And this (at least documentation doesn't say a thing) is pure, but under the `os` module /o\ This language is a mess…